docs/source/home.rst

```diff
@@ -23,7 +23,7 @@ you will always be able to interact with your environment, engage with your frie
               :target: explore/gallery.html
 
 
-In addition to the documentation, you can join the discussion on the Vircadia `Discord server <https://discordapp.com/invite/Pvx2vke>`__ |Discord|.
+In addition to the documentation, you can join the discussion on the Overte `Matrix <https://matrix.to/#/#overte:matrix.org>`_ or `Discord server <https://discord.gg/4YuQvc8K2f>`__ |Discord|.
```

Comment:
Should we really advertise the Discord bridge?
Discord will break compatibility again eventually, and there is already issues in bridging the two, such as messages being posted multiple times or messages disappearing.

Reply:
Thank you for the review!
I believe we should, at least for now. Using only Matrix would mean an additional barrier to entry. For example a lot of my friends don't have Matrix and are not interested in installing it. If bridge is causing problems we could add a note that Matrix is an official part and Discord is bridged.